content/python/concepts/casting/casting.md

```diff
@@ -5,23 +5,24 @@ Subjects:
   - 'Computer Science'
   - 'Data Science'
 Tags:
-  - 'Classes'
-  - 'Functions'
+  - 'Data Types'
+  - 'Variables'
+  - 'Variable Types'
 CatalogContent:
   - 'learn-python-3'
   - 'paths/computer-science'
 ---
 
-Casting is a method that converts a variable's data type into another data type. These conversions can be implicit (automatically compiled) or explicit (using built-in functions).
+Casting (also known as type conversion) is a process that converts a variable's data type into another data type. These conversions can be implicit (automatically interpreted) or explicit (using built-in functions).
 
 ## Implicit Type Conversion
 
 Python's interpreter automatically type casts one data type into another data type. This type of conversion is done without any user involvement.
 
-Since Python always convert smaller data type to higher data type to avoid data loss. In the below example, `7` a integer (lower data type) is type casted to a float (higher data type) when added with `2.2` to avoid data loss. So finally data type of y is `float`.
+In order to avoid data loss, Python converts smaller data types to higher data types. In the below example, the integer `7` is type casted to a float when added with `2.2`. Since the expression represents the sum of two `float` values, the data type of `y` is also `float`.
```

Comment:
```suggestion
In order to avoid data loss, Python converts smaller data types to higher data types. In the below example, the integer `7` is type casted to a `float` when added with `2.2`. Since the expression represents the sum of two `float` values, the data type of `y` is also `float`.
```

Reply:
Sorry to butt in on someone else's work here, but I believe the past tense of "cast" is "cast" — as with "broadcast" or when talking about an actor having been "cast" in a play or movie, for example. Therefore I think this should read:

>the integer `7` is type cast to a `float` when added with `2.2`.